src/test/java/org/apache/maven/plugins/dependency/tree/TestTreeMojo.java

```diff
@@ -127,6 +127,24 @@ public void _testTreeTGFSerializing() throws Exception {
         assertTrue(findString(contents, "testGroupId:release:jar:1.0:compile"));
     }
 
+    /**
+     * Test the JSON format serialization
+     */
+    public void _testTreeJsonSerialzing() throws Exception {
+        List<String> contents = runTreeMojo("tree1.json", "json");
+        assertTrue(findString(contents, "\"testGroupId\": \"project\""));
```

Comment:
not sure why findString is used here. contains is simple and more obvious

Reply:
I tried to keep the test case in the same style as the rest of the class. But I can change it.

Reply:
contains is clearer

Reply:
I want to double check: do you mean `contents.contains("...")`? 
Due to indentation the string might not be in the list of strings, but a substring of an element, which the `findString`-method seems to be created for in the same file.

Reply:
That makes sense.